src/main/java/hudson/plugins/ec2/AmazonEC2Cloud.java

```diff
@@ -201,6 +207,7 @@ public ListBoxModel doFillRegionItems(
                         model.add(name, name);
                     }
                 } catch (SdkClientException ex) {
+                    LOGGER.log(Level.INFO, "AmazonEC2Cloud.doFillRegionItems() got exception: " + ex);
```

Comment:
I was thinking I would change this message to incorporate the comment about this may happen before credentials are specified so maybe "OK".

Can we write unit tests which check for logging? The logging would have been VERY helpful to me initially when I encountered this issue.